src/app/components/Search/ResultsList.jsx

```diff
@@ -1,30 +1,29 @@
 import React from 'react';
+import { Link } from 'react-router';
 import {
   isEmpty as _isEmpty,
 } from 'underscore';
 import ResultsRow from './ResultsRow';
 
-const ResultsList = (results) => {
-  if (_isEmpty(results.results)) {
+const ResultsList = (props) => {
+  if (_isEmpty(props.results)) {
     return null;
   }
 
   return (
     <div>
       <h2>Works</h2>
       <ul className="nypl-results-list">
-        {
-          results.results.map((result, i) => (
+        {props.results.map((result, i) => (
             <li className="nypl-results-item" key={i.toString()}>
               <h3>
-                <a href={`/work/detail/${result._id}`}>
+                <Link to={{ pathname: '/work', query: { workId: `${result['_source'].ids[0].identifier}` } }}>
```

Comment:
It may be easier/safer to have this lookup works by UUID, which is always present and exists outside of the identifier array (and is guaranteed to be unique. While identifiers should uniquely identify works, I'm not 100% we can trust our sources).

If the single work endpoint doesn't support this right now I could potentially add that.

Reply:
This seems to be placeholder because the routes aren't fully set up yet. But I do agree about the UUID.

Reply:
Yes, a standard ID should be used -- definitely placeholder code to basically get used to using the <Link/> component of react-router. Will change to use the UUID property instead.